### DISCUSSION TITLE
cant display 3d using markerarray when i use frame id

### DISCUSSION THREAD

**Discussion by wangjunhe8127** [foxglove/mcap#1080] [General]
cant display 3d using markerarray when i use frame id
<img width="783" alt="image" src="https://github.com/foxglove/community/assets/63217254/198d8558-e8da-4bfa-9ef5-eed8108db9ad">

**Comment by jtbandes** [comment]
Could you please share an example file that we can use to reproduce the issue?

You could also try changing the dropdown from "ego" to "world" to match the marker.
